psscriptanalyzer/Backup-Database.psd1

```
@@ -0,0 +1,20 @@
+@{
+    RootModule        = 'Backup-Database.psm1'
```

PSScriptAnalyzer (1.25.0)

[warning] Use space before and after binary and assignment operators.

Suggested fix: Use space before and after binary and assignment operators.

(PSUseConsistentWhitespace)